src/cljs/replumb/repl.cljs

```diff
@@ -591,6 +593,24 @@
       (call-back (common/wrap-success (s/join (map #(with-out-str (repl/print-doc %)) ms))))
       (call-back (common/wrap-success "nil")))))
 
+(declare read-eval-call)
```

Comment:
I don't like this at all, it is better to actually extract the eval functionality as first class citizen not to call read-eval-call directly